example/lib/home_page.dart

```diff
@@ -88,7 +88,37 @@ class _MyHomePageState extends State<MyHomePage> {
               ),
             ),
             Text(
-              'Separated example:',
+              horizontalView
+                  ? 'Builder Cds example:'
+                  : 'Builder\nCds\nexample:',
+              style: Theme.of(context).textTheme.headlineSmall,
+            ),
+            Padding(
+              padding: const EdgeInsets.all(10),
+              child: SizedBox(
```

Comment:
When `width` and `height` are the same, prefer to use the `SizedBox.square` constructor